CHANGES.rst

```diff
@@ -6,6 +6,8 @@
 0.0.2 (unreleased)
 ==================
 
+- Metric kind `s` is now handled as a distinct type. See `issue 6
```

Comment:
Double backticks, please. Single backticks mean "default role", which in this case is object cross ref. Double backticks mean code.